tests/actions.go

```diff
@@ -363,6 +365,36 @@ func (oi *OperatorConfig) OperatorHelmSetString(m map[string]string) string {
 	return strings.Join(arr, ",")
 }
 
+func (oa *operatorActions) runKubectlOrDie(args ...string) string {
+	cmd := "kubectl"
+	glog.Infof("Running '%s %s'", cmd, strings.Join(args, " "))
+	out, err := exec.Command(cmd, args...).CombinedOutput()
```

Comment:
Wondering how kubectl communicate with api-server as there is no kube-config in e2e image?

Reply:
it runs in a pod, so kubectl will use in-cluster config. the service account belongs to `cluster-admin` role.

Reply:
thx.